scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
Maybe the later is better ( or `src/[nvim/]generators/`) as unlike most other stuff in `scripts/` they are not really meant to be invoked directly, rather as part of the build process with comandline composed cmake.